backend/src/services/auto_trader.py

```
@@ -33,61 +21,59 @@ def __init__(self, socketio=None, data_fetcher_service=None, ai_analysis_service
             self.socketio.on('rugpull_alert', self._handle_rugpull_alert)
-            logger.info("Registered rugpull_alert handler.")
 
     def _load_config(self) -> Dict:
         if os.path.exists(CONFIG_FILE):
             try:
                 with open(CONFIG_FILE, 'r') as f:
-                    config = json.load(f)
-                    logger.info(f"Loaded auto-trader config from {CONFIG_FILE}")
-                    return config
+                    return json.load(f)
             except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)